ZukoAzula2016/src/org/team1540/zukoazula/DriveCode.java

```diff
@@ -25,19 +26,25 @@
 
     private static final FloatIO driveEncoder = leftCANs[0].modEncoder().getEncoderPosition();
 
-    private static final ArbitratedFloat leftInput = ZukoAzula.behaviors.addFloat();
-    private static final ArbitratedFloat rightInput = ZukoAzula.behaviors.addFloat();
+    private static final BehaviorArbitrator behaviors = new BehaviorArbitrator("Behaviors");
+    private static final Behavior autonomous = behaviors.addBehavior("Autonomous", FRC.inAutonomousMode());
```

Comment:
While this and the line below is only currently used in DriveCode, autonomous and teleop _might_ be used elsewhere (in theory,) and since they are in no way _specific_ to DriveCode, I disagree with you moving at least autonomous and teleop here.